docs/features.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Supported features
+sidebar_label: Features
+---
+
+Margarita is heavily inspired by our <a href="http://kiwi.com" target="_blank">Kiwi.com</a> mobile and web apps. We want to create full-fledged production ready app for developers, entrepreneurs and companies who want to use our API.
+
+## Supported Features
+
+Margarita is separated to 4 main parts of the app:
+
+- [Search for connections](features#search-for-connections)
+- [Booking](features#booking)
+- [Manage my booking](features#manage-my-booking)
+- [Payments](features#payments)
+
+### Search for connections
+
+Get an access to the database of 15 billion connections virtually interlined by <a href="http://kiwi.com" target="_blank">Kiwi.com</a>.
+
+#### Search form
+
+- Initial screen of whole app. User can set here all options about connection he is looking for.
+
+<div>
+<img src="./assets/screens/search.png" alt="Search" title="Search" width="250"/>
+<img src="./assets/screens/search-passenger.png" alt="" title="" width="250"/>
+<img src="./assets/screens/search-trip-type.png" alt="" title="" width="250"/>
+</div>
+
+#### Calendar
+
+- User can choose if he wants single date, range date or specify how many night he wants to stay in selected destination.
+
+<div>
+<img src="./assets/screens/range-date-picker.png" alt="" title="" width="250"/>
+<img src="./assets/screens/range-date-picker-nights.png" alt="" title="" width="250"/>
+</div>
+
+#### Place Picker
+
+- We use Tequila for search and autocomplete places in input field. It's also possible to add multiple places for origin and destination.
```

Comment:
We use Tequila for searching and also showing place suggestions in the input field when inputting places.

Reply:
Done